app-center-webapps/src/main/webapp/skin/less/app-center.less

```diff
@@ -485,230 +485,229 @@
+        }
+        input:focus {
+          outline: none!important;
+        }
+        i {
+          color: @greyColorLighten1Default!important;
+        }
       }
-    }
-    #appCenterAdminSetup {
-      align-self: center;
-      i {
-        color: @greyColorLighten1Default!important;
+      #appCenterAdminSetup {
+        align-self: center;
+        i {
+          color: @greyColorLighten1Default!important;
+        }
       }
     }
   }
 }
-.VuetifyApp .userFavoriteApplications {
-  padding: 14px;
+.VuetifyApp .userFavoriteApplicationsContainer {
   border-radius: 4px;
-  flex: 1 1 23%;
   background: @baseBackgroundDefault;
+  
+  .userFavoriteApplications {
+    padding: 14px;
 
-  .favoriteAppTitle {
-    width: 100%;
-    color: #6d6d6d;
-    font-size: 20px;
-    font-weight: 500;
-    letter-spacing: 1px;
-    padding-top: 25px;
-    padding-bottom: 35px;
-  }
-  .favoriteApplication {
-    border-bottom: 1px solid #efefef;
-    display: inline-flex;
-    margin-bottom: 15px;
-    width: 100%;
-    border-radius: 8px!important;
-    .favoriteAppImage {
-      .appImage {
-        box-shadow: 0 0 5px darkgrey;
-        object-fit: cover;
-        overflow: hidden;
-      }
-      .default {
-        border-radius: unset;
-        box-shadow: none;
-        margin-top: -53px;
-        margin-left: 20px;
-      }
+    .favoriteAppsTitle {
+      width: 100%;
+      color: @darkGrey;
+      font-size: 20px;
+      font-weight: 500;
+      letter-spacing: 1px;
+      padding-top: 25px;
+      margin-bottom: 35px;
     }
-    .favoriteAppUrl {
-      max-width: 130px;
-      h5 {
-        color: #797979;
-        font-size: 15px;
-        padding-left: 10px;
-        flex: 1;
+    .favoriteApplication {
+      border-bottom: 1px solid #efefef;
+      display: inline-flex;
+      margin-bottom: 15px;
+      width: 100%;
+      border-radius: 8px!important;
+      .favoriteAppImage {
+        .appImage {
+          box-shadow: 0 0 5px darkgrey;
+          object-fit: cover;
+          overflow: hidden;
+        }
+        .default {
+          border-radius: unset;
+          box-shadow: none;
+          margin-top: -53px;
+          margin-left: 20px;
+        }
+      }
+      .favoriteAppUrl {
+        h5 {
+          color: @textColorDefault;
+          font-size: 15px;
+          padding-left: 10px;
+          flex: 1;
+
+          .favAppTitle {
+            max-width: 196px;
+            white-space: nowrap;
+            overflow: hidden;
+            text-overflow: ellipsis;
+          }
+        }
       }
     }
-    .favoriteAppRemove a i:before {
+    .maxFavorite {
+      color: @greyColorLighten1Default;
+      font-style: italic;
       font-size: 14px;
-      font-weight: normal;
-      color: #929292;
     }
-  }
-  .maxFavorite {
-    color: @greyColorLighten1Default;
-    font-style: italic;
-    font-size: 14px;
-  }
-  .reached {
-    color: #E53935;
+    .reached {
+      color: #E53935;
```

Comment:
Can we use a variable ?

Reply:
Used in platform-ui but no variable for this color.